fake.go

```diff
@@ -34,6 +34,8 @@ type Fake struct {
 	// Table contains the Interface's table. This will be `nil` until you `tx.Add()`
 	// the table.
 	Table *FakeTable
+
+	LastTx *Transaction
```

Comment:
// LastTx is the last transaction passed to Run(). It will remain set until the
    // next time Run() is called. (It is not affected by Check().)

Hm... and maybe `LastTransaction` though? None of the other (exported) field names are abbreviated...